server/simple/public/index.html

```diff
@@ -34,21 +35,24 @@
 </style>
 
 <body>
-<h1>Welcome to Dashland!</h1>
+<h1>pkg:shelf example</h1>
 <img src="/dashland.jpeg" alt="Dashland">
```

Comment:
I wonder... should we put the image below the text? It seems like it might push the text below the fold on a laptop screen. (But if it doesn't, lgtm.)

Reply:
🤷  – I don't want to get too fancy here. This is just the content IN the demo.